tutorials/Quantile_Interval_And_Interval_Scores.ipynb

```diff
@@ -8,33 +8,42 @@
     "\n",
     "The quantile interval and interval scores are consistent scoring functions to evaluate prediction intervals, where the aim is to predict an interval that captures uncertainty at a given confidence level. Quantile interval score can accept any values for the lower and upper quantile levels as long as $0 < \\text{lower quantile level} < \\text{upper quantile level} < 1$ is met, whereas interval score is only for forecast intervals with symmetric quantile level range. For example lower and upper quantile levels of 0.2 and 0.9 can be used by the quantile interval score but not by the interval score.\n",
     "\n",
-    "The quantile interval score function is defined as sum of three penalties, as follows:\n",
+    "The quantile interval score function is defined as the sum of three penalties, as follows:\n",
     "\n",
     "- **Interval Width Penalty:**\n",
     "$$ W = q_{u} - q_{l} $$\n",
     "- **Over-Prediction Penalty:**\n",
-    "$$ U = \\begin{cases} \n",
+    "$$ O = \\begin{cases} \n",
     "\\frac{1}{\\alpha_l} \\cdot (q_{l} - y) & \\text{if } y < q_{l} \\\\\n",
     "0 & \\text{otherwise}\n",
     "\\end{cases} $$\n",
     "- **Under-Prediction Penalty:**\n",
-    "$$ O = \\begin{cases} \n",
+    "$$ U = \\begin{cases} \n",
     "\\frac{1}{1 - \\alpha_u} \\cdot (q_{u} - y) & \\text{if } y > q_{u} \\\\\n",
     "0 & \\text{otherwise}\n",
     "\\end{cases} $$\n",
-    "So, the total score can be written as:\n",
-    "$$ S = W + U + O $$\n",
+    "**So, the total score can be written as:**\n",
+    "$$ S = W + O + U $$\n",
     "\n",
-    "where $S$ is the scoring function (here *quantile interval score*), $q_l$ and $q_u$ are lower and upper quantile forecasts, and $y$ is observation, $\\alpha_l$ and $\\alpha_u$ are lower and upper quantile level. Lower values of the quantile interval scores are better. As you can see, this score penalizes the width of interval as well as the extent to which observation falls outside of the interval forecast (i.e., under-prediction penalty when observation is larger than upper-quantile forecast, and over-prediction penalty when observation is smaller than lower-quantile forecast).\n",
+    "where:\n",
+    "- $S$ is the scoring function (here quantile interval score),\n",
```

Comment:
The dotpoints are not rendering correctly in readthedocs. The screenshot below shows how they are currently rendering in readthedocs:
![image](https://github.com/user-attachments/assets/7f020281-5dc2-49c8-9167-586c2ccecef1)

This can usually be fixed by adding an empty line before the first dot point.